recipe/migrations/r-base44_and_m2w64-ucrt.yaml

```diff
@@ -9,6 +9,12 @@ __migrator:
   longterm: True
   include_noarch: True
   pr_limit: 4
+  override_cbc_keys:
+    - r-base
+    - r_base
```

Comment:
Do we mean to have both `-` and `_`? Thought these evaluate to the same thing

Reply:
It's equivalent to https://github.com/regro/cf-scripts/blob/ee87be96ed34c44a576a0a4b456dd00bd381695c/conda_forge_tick/make_migrators.py#L439 which is only done for the non-`override_cbc_keys` case.